rapid7_insightvm/komand_rapid7_insightvm/actions/get_expiring_vulnerability_exceptions/action.py

```diff
@@ -0,0 +1,43 @@
+import komand
+from .schema import GetExpiringVulnerabilityExceptionsInput, GetExpiringVulnerabilityExceptionsOutput, Input, Output, Component
+# Custom imports below
+from komand_rapid7_insightvm.util import endpoints
+from komand_rapid7_insightvm.util.resource_helper import ResourceHelper
+import maya
+
+class GetExpiringVulnerabilityExceptions(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+                name='get_expiring_vulnerability_exceptions',
+                description=Component.DESCRIPTION,
+                input=GetExpiringVulnerabilityExceptionsInput(),
+                output=GetExpiringVulnerabilityExceptionsOutput())
+
+    def run(self, params={}):
+        days_left = params.get(Input.EXPIRES_IN_LESS_THAN)
+
+        params = {
+            "sort": "expires,ASC"
+        }
+
+        # Get all the exceptions
+        resource_helper = ResourceHelper(self.connection.session, self.logger)
+        endpoint = endpoints.VulnerabilityException.vulnerability_exceptions(self.connection.console_url)
+        response = resource_helper.resource_request(endpoint=endpoint, method='get', params=params)
+        resources = response.get("resources", [])
+
+        now = maya.now()
+        exceptions_to_return = []
+
+        # This looks for any exceptions that are expiring sooner than the
+        # days specified by the user. It appends any that are to the output array.
+        for i, exception in enumerate(resources):
+            expires = exception.get("expires", "")
```

Comment:
```suggestion
            expires = exception.get("expires")
```
This can default to None based on the check on line 37